### PR TITLE
Add tooltips and example entries to Edit Project and My Profile pages

### PR DESCRIPTION
**Pull request by katyaivaniuk**
## Description

1. Implemented tooltips on the Edit Project and My Profile pages.
2. Added a link to an example project on the Edit Project page and updated the /home page to extract the optional id parameter for automatically opening the project. (Note: This is used only when redirecting from the Edit Project page.)
3. Revised titles, placeholders, and added descriptive tooltips to provide clearer guidance on the information users need to enter.
4. When exploring other projects, the entries are not in the "asking form" "Provide a link to the project", but rather statement form like "Project URL" 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Tests (adding or updating tests)
- [ ] Documentation update

## Testing Instructions

1. Check if you have the same visual representation of the tooltips and example placeholders. 
2. Confirm that clicking on the "here" link on the Edit Project page redirects you to the correct page, displaying the exemplar project titled "SeatMagician" created by Ava, along with the feedback.

Please note that while clicking "here" on the Edit Project page correctly redirects to the appropriate project ID in production, you will need to change the domain to localhost:3000/home?id=... for local testing, as the production version of the home page does not yet support the logic for automatically opening the project.

## Screenshots (if applicable)
telegram-cloud-photo-size-2-5393567738330491221-y

telegram-cloud-photo-size-2-5393567738330491222-y

telegram-cloud-photo-size-2-5393567738330491223-y

telegram-cloud-photo-size-2-5393567738330491225-y

If your changes include visual components, add screenshots showing the affected pages or elements.

### CODE REVIEW

File: src/app/project_idea/page.tsx [hunk 1]
```diff
@@ -61,7 +61,15 @@ export default async function ProjectIdeaScreen({
 					}}
 				>
 					Fill out as much detail as possible to help others understand your
-					project.
+					project. You can view the example entry{" "}
+					<a
+						href="https://id8.guru/home?id=50e834ba-12d5-466e-8734-42a54487f5a7"
```
Comment (Mykhailo-Chudyk) [inline src/app/project_idea/page.tsx:66]: This should work for now, but ideally, we should use the environment variable to direct users to either `localhost` or `id8.guru`

File: src/components/PersonalDetailsForm.tsx [hunk 2]
```diff
@@ -177,78 +185,228 @@ const PersonalDetailsForm: React.FC<PersonalDetailsFormProps> = ({
...
-					</LocalizationProvider>
-					<TextField
-						fullWidth
-						label="Gender"
-						value={gender}
-						onChange={(e) => setGender(e.target.value)}
-						variant="outlined"
-					/>
-					{/* disabled for now */}
-					{/* <TextField
```
Comment (Mykhailo-Chudyk) [inline src/components/PersonalDetailsForm.tsx:204]: Good that you deleted this :)

**Review by Mykhailo-Chudyk — APPROVED** [review]
LGTM!

File: src/app/project_idea/page.tsx (hunk 1 above)
Comment (katyaivaniuk) [reply src/app/project_idea/page.tsx:66]: Sounds good!

**Pull request merged**